ci/set_port_range.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# See http://mesos.apache.org/documentation/latest/port-mapping-isolator/
```

Comment:
404 for me. 
found https://mesos.readthedocs.io/en/latest/port-mapping-isolator/

Reply:
Good find. Seems the link is outdated. This is a copy from Marathon.

Reply:
```suggestion
# See https://mesos.readthedocs.io/en/latest/port-mapping-isolator/
```